pkg/controller/jobframework/integrationmanager.go

```diff
@@ -136,6 +137,36 @@ func RegisterIntegration(name string, cb IntegrationCallbacks) error {
 	return manager.register(name, cb)
 }
 
+func RegisterExternalIntegration(name string) error {
+	splitName := strings.Split(name, ", Kind=")
```

Comment:
I think the version should not matter. And you can use this function https://github.com/kubernetes/apimachinery/blob/37988e577e160bf7749437141fee1d6b8ca15dd6/pkg/runtime/schema/group_version.go#L76